app/views/reports/change_report.pdf.erb

```diff
@@ -21,17 +21,18 @@
 
 <div class="divider"></div>
 
-<div class="spacer"></div>
-
-<% @report.reported_changes.each do |change| %>
-  <%= render partial: 'reports/member_info', locals: {member: change.member} %>
-  
+<% @changes_grouped_by_member.each do |change| %>
+  <%= render partial: 'reports/member_info', locals: {member: MemberDecorator.new(change.member)} %>
   <div class="divider"></div>
-  
   <%= render partial: "reports/change", locals: {change: ChangeDecorator.new(change)} %>
-  <div class="page-break"></div>
+
+  <% unless change.equal?(@changes_grouped_by_member.last) %>
+    <div class="page-break"></div>
```

Comment:
This unless statement is to have it so the page-break doesn't occur on the last change page.